src/store/reducers/payees.ts

```diff
@@ -0,0 +1,40 @@
+import { clearPayeeForm, loadPayees, removePayee, setPayee, updatePayeeForm } from '@actions'
+import { RootAction } from '@store'
+import { Payee } from '@types'
+import { combineReducers } from 'redux'
+import { getType } from 'typesafe-actions'
+
+export interface PayeeState {
+  form: Payee
+  payees: Payee[]
+}
+
+export const payees = combineReducers<PayeeState, RootAction>({
+  form: combineReducers<Payee, RootAction>({
+    name: handleChange('name'),
+    publicKey: handleChange('publicKey'),
+  }),
+  payees: (state = [], action) => {
+    switch (action.type) {
+      case getType(setPayee):
+        return state.concat(action.payload)
```

Comment:
This is a redux no-no. `{...state, {...action.payload}}`

Reply:
or just `{...state, action.payload}`

Reply:
concat returns a new array, it doesn't mutate, so I disagree. But! updated

Reply:
Yeah I wrote this and was like, wait I think this is okay.